audit/src/main/java/com/ritense/audit/autoconfigure/AuditAutoConfiguration.java

```diff
@@ -76,4 +84,18 @@ public AuditEventListener auditEventListener(AuditEventProcessor auditEventProce
         return new AuditEventListenerImpl(auditEventProcessor);
     }
 
+    @Bean
+    @ConditionalOnProperty(prefix = "valtimo", name = "database", havingValue = "postgres")
+    public JpaRepositoryFactoryBean<AuditRecordRepository<AuditRecord, AuditRecordId>, AuditRecord, AuditRecordId> postgresAuditRecordRepository() {
```

Comment:
Waar is die JpaRepositoryFactoryBean voor nodig?

Reply:
Dat is de manier om in spring een implementatie van een repository interface op te geven. Je kan niet `new SomethingRepository()` doen omdat het een interface is. Zo kan je dus kiezen wel DBMS specifieke repository er gebruikt wordt.